datarepo-clienttests/src/main/java/scripts/deploymentscripts/LaunchLocalProcess.java

```diff
@@ -77,8 +77,7 @@ public void deploy(ServerSpecification server, ApplicationSpecification app) thr
     UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
     // call the unauthenticated status endpoint
     try {
-      unauthenticatedApi.serviceStatus();
```

Comment:
👍 I was wondering about this being on two lines.

Reply:
Two lines -- you mean before? It was only calling the API endpoint once, and the second line was to get the status code. New client can do it all in one line!